src/FormElement/ScheduleElement/MonthlyFields.php

```diff
@@ -166,4 +162,30 @@ protected function registerAttributeCallbacks(Attributes $attributes)
             ->registerAttributeCallback('availableFields', null, [$this, 'setAvailableFields'])
             ->registerAttributeCallback('protector', null, [$this, 'setIdProtector']);
     }
+
+    protected function addDefaultValidators(ValidatorChain $chain): void
+    {
+        $chain->add(
+            new CallbackValidator(function ($_, CallbackValidator $validator): bool {
+                if ($this->getPopulatedValue('runsOn', static::RUNS_EACH) !== static::RUNS_EACH) {
```

Comment:
Why `getPopulatedValue` here and `getValue` below?

Reply:
Makes no difference!

Reply:
Then please use `getValue` exclusively.